### PR TITLE
Use angle bracket syntax for amount-input

### PR DESCRIPTION
**Pull request by mrloop**
#### Description
[//]: # "Please include a summary of the change. Imagine you're trying to explain the changes to a reviewer."
 
Update tests, docs and comments to use angle bracket syntax

### CODE REVIEW

**Review by daltino — APPROVED** [review]

File: tests/dummy/app/templates/docs/components/amount-input.md [hunk 1]
```diff
@@ -15,9 +16,10 @@
 
 <DocsDemo as |demo|>
   <demo.example @name="amount-input-disabled">
-    {{amount-input
-      value=924.67
-      disabled=true }}
+    <AmountInput
+      @value=924.67
```
Comment (zeppelin) [inline tests/dummy/app/templates/docs/components/amount-input.md:20]: `typeof this.args.value === 'string'`

File: tests/dummy/app/templates/docs/components/amount-input.md [hunk 2]
```diff
@@ -26,13 +28,14 @@
...
-    {{amount-input
-      placeholder="1.000.000"
-      step=1
-      currency="USD"
-      numberOfDecimal=0
-      value=valueParams
-      update=(action (mut valueParams)) }}
+    <AmountInput
+      @placeholder="1.000.000"
+      @step=1
```
Comment (zeppelin) [inline tests/dummy/app/templates/docs/components/amount-input.md:33]: `typeof this.args.value === 'string'`

File: tests/dummy/app/templates/docs/components/amount-input.md [hunk 3]
```diff
@@ -26,13 +28,14 @@
...
+      @currency="USD"
+      @numberOfDecimal=0
```
Comment (zeppelin) [inline tests/dummy/app/templates/docs/components/amount-input.md:35]: `typeof this.args.value === 'string'`

**Review by zeppelin — CHANGES_REQUESTED** [review]
There are a bunch of component invocations that are passing numbers to args. Is it intentional? If so, wouldn't it be better to mark them explicitly as string with quotes? Same goes for booleans, It'd be likely better to wrap them to curlies.

**Comment by mrloop** [comment]
> There are a bunch of component invocations that are passing numbers to args. Is it intentional?

No an oversight 🙈 have updated

**Pull request merged**